### DISCUSSION TITLE
riskfirst.org

### DISCUSSION THREAD

**Discussion by robmoffat** [risk-first/website#64] [Team Posts]
Github wikis leave a lot to be desired.  Submitting PRs against them being a big one, and the fact that the content can be viewed from different URLs, potentially meaning images don't load.

Github provides _Github Pages_ which allows you to host a static site through them, using Jekyll as a templating engine. They provide various templates you can use, and failing that you can download others or customize something yourself.  Additionally, if you own a domain, you can point this to the site.

Hence, for a while I've had a placeholder Github Pages site at [riskfirst.org](https://riskfirst.org).

Over the last couple of days, I've worked on improving the in-built theme (minimal) and migrating the content over.   I'd appreciate some feedback, let me know if it's easy to read/looks ok!  (I've only looked on Chrome/Safari...)

One of the concerns I've had about doing this is that I would end up with content duplicated on the github wiki and on the pages site.  I've fixed this by adding placeholder redirects on the wiki side, so that people can find the page they were after (just a bit of bash scripting).  Hopefully this will mean search engines link to the right page correctly.  

Did I do it ok?

thanks,
Rob

**Comment by mitranc** [comment]
Hi Rob,

Check how I'm using Travis integration icons on my opensource fix engine.
There is a Build status on the Readme.md depicting a way of linking
external images.
A PR on the readme would be previewable this way.

Haven't tried it with wiki's but the trick should work there too.

https://github.com/mitranc/scala-fix-engine


Thanks,
C

**Comment by robmoffat** [comment]
Like a build status indicator, you mean?

Yeah there must be some kind of automated build going on at the github end.   I occasionally get emails like this:

<img width="532" alt="screenshot 2019-01-24 at 17 23 34" src="https://user-images.githubusercontent.com/568673/51696279-05712980-1ffd-11e9-93be-c767f61088a9.png">

... which tell me I screwed something up.  But otherwise, it takes about a minute to refresh the gh-pages site.  There doesn't seem to be a status indicator anywhere I can see.
